tests/system/import_test.go

```diff
@@ -19,16 +22,57 @@ func TestImport(t *testing.T) {
 		t.Fatalf("Failed to read dockerrun: %v", err)
 	}
 
-	log.Debugf("Creating test resources")
-	environment := s.Layer0.CreateEnvironment("import")
-	loadBalancer := s.Layer0.CreateLoadBalancer("sts", environment.EnvironmentID)
-	deploy := s.Layer0.CreateDeploy("sts", data)
-	service := s.Layer0.CreateService("sts", environment.EnvironmentID, deploy.DeployID, loadBalancer.LoadBalancerID)
+	log.Printf("[DEBUG] Creating test resources")
+	createEnvironmentReq := models.CreateEnvironmentRequest{
+		EnvironmentName:  "import",
+		InstanceSize:     "m3.medium",
```

Comment:
[nit] I've been noticing that `t2.micro` instances start up faster, and they reduce the spend slightly. Seems more appropriate for using with tests, at least.

Reply:
:white_check_mark: